osidb/serializer.py

```diff
@@ -621,6 +622,31 @@ def to_representation(self, value):
         return list(trackers)
 
 
+class FlawReferenceSerializer(
+    ACLMixinSerializer,
+    BugzillaSyncMixinSerializer,
+    TrackingMixinSerializer,
+):
+    """FlawReference serializer"""
+
+    class Meta:
+        """filter fields"""
+
+        model = FlawReference
+        fields = (
+            ["description", "flaw", "type", "url", "uuid"]
+            + ACLMixinSerializer.Meta.fields
+            + TrackingMixinSerializer.Meta.fields
+        )
+
+
+@extend_schema_serializer(exclude_fields=["updated_dt"])
+class FlawReferencePostSerializer(FlawReferenceSerializer):
+    # extra serializer for POST request as there is no last update
+    # timestamp but we need to make the field mandatory otherwise
+    pass
```

Comment:
this wouldn't be necessary if `updated_dt` was a simple `read_only=True` field, not that there's anything you can do here but see my comment at https://github.com/RedHatProductSecurity/osidb/pull/165#issuecomment-1559125237